src/loaders/tvg/tvgTvgLoader.cpp

```diff
@@ -70,6 +70,7 @@ bool TvgLoader::readHeader()
     ptr += SIZE(float);
 
     //4. Reserved
+    if (!(*ptr & TVG_HEAD_FLAG_VALIDATION)) return false;
     if (*ptr & TVG_HEAD_FLAG_COMPRESSED) compressed = true;
```

Comment:
We already put the signature of the file (the minimum checking for the file type), this kind of similar validation flag is just duplicated in its purpose, not prevent the error case 100%. If you think validation is necessary, then we should introduce the perfect validation check such as CRC(Cyclic Redundancy Check)